src/utils/HttpClient.js

```diff
@@ -1,66 +0,0 @@
-import request from 'superagent';
-import { canUseDOM } from 'fbjs/lib/ExecutionEnvironment';
-
-function getUrl(path) {
```

Comment:
I think this can be fixed by moving this logic to an `axios.interceptor` which looks something like: 

```javascript
// Add a request interceptor
axios.interceptors.request.use(function (config) {
    // Do something before request is sent
    return config;
  }, function (error) {
    // Do something with request error
    return Promise.reject(error);
  });
```